.github/workflows/gh-pages.yml

```diff
@@ -20,12 +20,13 @@ jobs:
 
       - name: Install and Build 🔧
         run: |
+          cd playground
           npm install
-          npm run preinstall
+          npm run build:miden
           npm run build
           npx webpack
       - name: Deploy to GitHub Pages
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: build
\ No newline at end of file
+          publish_dir: build
```

Comment:
Let's keep the newline at the end of the file

Reply:
done